src/pages/mission/entry/components/MissionInfoCard.tsx

```
@@ -0,0 +1,141 @@
+  };
+
+  return (
+    <div className="flex w-full flex-col gap-8 rounded-xl bg-white px-23 py-21">
+      <div className="flex items-center gap-10">
+        <h2 className="heading-3 pb-4 text-center text-black">미션 이름</h2>
+      </div>
+      <div className="flex items-start gap-4 self-stretch pb-8">
+        {keywords.map((keyword, index) => (
+          <div
+            key={index}
+            className="body-4 flex items-center justify-center gap-4 rounded-lg bg-moamoa-50 px-16 py-8 text-center text-moamoa-500"
```

Biome (2.3.13)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)